src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+
+type IntrinsicElements = JSX.IntrinsicElements["input"];
+
+interface Props extends IntrinsicElements {
+  checked?: boolean;
+  children?: JSX.Element[];
+  id: string;
+  isTile?: boolean;
+  label: string;
+  name: string;
+  tileDescription?: string;
+  value?: string;
+}
+
+/**
+ * Checkbox Component
+ * @param {string}  id                A unique identifier for the input.
+ * @param {string}  label             Label text that appears to the right of the Checkbox.
+ * @param {string}  name              Name of the input element.
+ * @param {boolean} [checked]         Set the initial checked state.
+ * @param {Array}   [children]        An array of child elements to appear when checked.
+ * @param {boolean} [isTile]          Use the tile variation of the Checkbox.
+ * @param {string}  [tileDescription] Text that can be used to describe the label in more detail. Activates the tile variation automatically.
+ * @param {string}  [value]           Value of the input element.
+ */
```

Comment:
I know that disabled is coming from IntrinsicElements, but I'm thinking there might be some value to exposing that property to storybook?

Reply:
Sounds good to me 👍🏻